packages/dds/tree/src/feature-libraries/forestIndex.ts

```diff
@@ -33,14 +38,13 @@ import { jsonableTreeFromCursor } from "./treeTextCursor";
 export class ForestIndex implements Index<unknown>, SummaryElement {
     readonly key: string = "Forest";
 
-    // TODO: implement this to provide snapshots in summaries.
     readonly summaryElement?: SummaryElement = this;
 
     private readonly cursor: ITreeSubscriptionCursor;
 
     // Note that if invalidation happens when these promises are running, you may get stale results.
-    private readonly treeBlob: ICachedValue<Promise<ISummaryAttachment>>;
-    private readonly schemaBlob: ICachedValue<Promise<ISummaryAttachment>>;
+    private readonly treeBlob: ICachedValue<Promise<IFluidHandle<ArrayBufferLike>>>;
+    private readonly schemaBlob: ICachedValue<Promise<IFluidHandle<ArrayBufferLike>>>;
```

Comment:
This is probably a @CraigMacomber question, but what kind of schema data is stored in a ForestIndex?